services/drupal/web/modules/custom/epa_workflow/src/Plugin/Block/EpaNodeDetailsBlock.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+namespace Drupal\epa_workflow\Plugin\Block;
+
+use Drupal\Core\Block\BlockBase;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Link;
+use Drupal\Core\Url;
+
+/**
+ * Provides an EPA Node Details block.
+ *
+ * @Block(
+ *   id = "epa_node_details",
+ *   admin_label = @Translation("EPA Node Details"),
+ *   category = @Translation("Custom"),
+ *   context_definitions = {
+ *     "node" = @ContextDefinition("entity:node", required = TRUE, label = @Translation("Node"))
+ *   }
+ * )
+ */
+class EpaNodeDetailsBlock extends BlockBase implements ContainerFactoryPluginInterface {
```

Comment:
@julianamincey One other thing we'll want to have is a `blockAccess()` method to ensure the block only shows for the intended users. If you look on integration now the block shows up for anonymous users.

Take a look at https://github.com/USEPA/webcms/blob/FEATURE-epa-node-tabs/services/drupal/web/modules/custom/epa_workflow/src/Plugin/Block/FollowWidgetBlock.php#L104.

Rather than checking if the user has a specific permission like the FollowWidget,  this block we should only allow access if the user is authenticated. Sorry for not catching this earlier.